waifus/madoka yabuuchi

```diff
@@ -0,0 +1,11 @@
+{
+	"type" : 0,
+	"english-name" : "Madoka yabuuchi",
+	"image" : "https://cdn.myanimelist.net/images/characters/8/379321.jpg",
+	"japanese-name" : "藪内 円",
+	"year" : 2019,
+	"month" : 08,
+	"day-of-month" : 31,
+	"age-in-show": 16,
```

Comment:
Same thing here, you don't have any sources that say that she's 16 during the show.